models/userModel.js

```diff
@@ -0,0 +1,66 @@
+const { Model , DataTypes } = require('sequelize'); // requiring sequelize
+const bcrypt = require('bcrypt'); // requiring for encryption
+//const sequelize = require('');
```

Comment:
const sequelize = require('../config/connection');

This is where the DB connection happens.